ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateSymbolStylesFromWebStyles/CreateSymbolStylesFromWebStyles.qml

```diff
@@ -0,0 +1,227 @@
+// [WriteFile Name=CreateSymbolStylesFromWebStyles, Category=DisplayInformation]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.11
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+    property var symbolNames: []
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            id: map
+            Basemap {
+                initStyle: Enums.BasemapStyleArcGISNavigation
+            }
+
+            // Set the scale at which feature symbols and text will appear at their default size
+            referenceScale: 100000
+
+            FeatureLayer {
+                id: webLayer
+                ServiceFeatureTable {
+                    url: "http://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/LA_County_Points_of_Interest/FeatureServer/0"
+                }
+
+                UniqueValueRenderer {
+                    id: uniqueValueRenderer
+
+                    // The UniqueValueRenderer will affect specific features based on the values of the specified FieldName(s)
+                    fieldNames: ["cat2"]
+                }
+
+                Component.onCompleted: {
+                    const symbolKeys = ["atm", "beach", "campground", "city-hall", "hospital", "library", "park", "place-of-worship", "police-station", "post-office", "school", "trail"];
+                    let symbolsFetchedCount = 0;
+
+                    symbolKeys.forEach((symbolKey) => {
+                                           var symbolStyle = ArcGISRuntimeEnvironment.createObject("SymbolStyle", {styleName: "Esri2DPointSymbolsStyle", portal: {}});
```

Comment:
As mentioned on the C++ sample above, I think it would be better to declare 1 SymbolStyle instance, setup 1 signal handler, and call fetchSymbol on that one instance for each key. One potential tricky part is that our QML API will only let you run one fetchSymbol at a time, so you will need to queue them up - C++ will allow you to run several concurrently.